spec/helpers.lua

```diff
@@ -2797,6 +2800,11 @@ end
 -- @field mock_upstream_stream_port
 -- @field mock_upstream_stream_ssl_port
 -- @field mock_grpc_upstream_proto_path
+-- @field grpcbin_host The host for grpcbin service
+-- @field grpcbin_port The port (SSL disabled) for grpcbin service
+-- @field grpcbin_ssl_port The port (SSL enabled) for grpcbin service
+-- @field grpcbin_url The URL (SSL disabled) for grpcbin service
+-- @field grpcbin_ssl_url The URL (SSL enabled) for grpcbin service
```

Comment:
can we add the name of the env variable to set it to the documentation?

Reply:
probably should be added for the existing entires as well

Reply:
Do you mean stating the name of the environment variable in the comment?

Reply:
yes, such that the docs make clear how to set it.

for generating docs do `cd spec && ldoc . && open docs/index.html && cd ..` (requires `ldoc` to be installed)

Reply:
done